staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -428,6 +428,7 @@ const (
 	ApplySet              FeatureGate = "KUBECTL_APPLYSET"
 	ExplainOpenapiV3      FeatureGate = "KUBECTL_EXPLAIN_OPENAPIV3"
 	CmdPluginAsSubcommand FeatureGate = "KUBECTL_ENABLE_CMD_SHADOW"
+	AutoServerSide        FeatureGate = "KUBECTL_AUTO_SERVER_SIDE"
```

Comment:
Super nit (naming nit) - should we include "APPLY" somewhere in the value the user specifies?  `KUBECTL_APPLY_SERVER_SIDE_AUTO` or `KUBECTL_AUTO_SERVER_SIDE_APPLY` ?